recipes/resiprocate/all/conanfile.py

```diff
@@ -51,12 +60,17 @@ def _configure_autotools(self):
         configure_args = [
             "--enable-shared={}".format(yes_no(self.options.shared)),
             "--enable-static={}".format(yes_no(not self.options.shared)),
-            "--with-ssl={}".format(yes_no(not self.options.with_ssl)),
-            "--with-mysql={}".format(yes_no(not self.options.with_mysql)),
-            "--with-postgresql={}".format(yes_no(not self.options.with_postgresql)),
-            "--with-pic={}".format(yes_no(not self.options.fPIC))
+            "--with-pic={}".format(yes_no(self.options.fPIC))
```

Comment:
```suggestion
            "--with-pic={}".format(yes_no(self.options.get_safe("fPIC")))
```

Reply:
The default must be True

```suggestion
            "--with-pic={}".format(yes_no(self.options.get_safe("fPIC", True)))
```

Reply:
`get_safe` seems nice. But @madebr, I don't think the default should be True. If fPIC is deleted, then the default value True will be passed, which ends up adding `--with-pic` option in configuration.

Reply:
All code must be compiled with pic, when linked inside a shared library.

Reply:
Ok I got it. I just tried building a shared library without pic and I saw you are right. It doesn't build, and it needs pic option. 

It is quite confusing for me to delete fPIC option, while passing --with-pic option to configuration. Anyways, I will work on the code according to your comments.

Reply:
You can just leave out the `--with-pic` option. The `AutotoolsBuildEnvironment` handles `self.options.fPIC` automatically.

The reason we remove it, is because `pic` doesn't do anything on Windows,
and non-pic objects/libraries can't be used inside a shared library.